_data/projects/emby.yml

```diff
@@ -0,0 +1,15 @@
+name: Emby
+desc: The open media solution for your personal videos, music, photos, and Live TV.
+site: https://github.com/MediaBrowser/Emby
+tags:
+- ".NET"
+- javascript
+- C#
+- IOS
+- Android
+- Windows Phone
+- Windows 10
+- Roku
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/MediaBrowser/Emby/labels/up%20for%20grabs
```

Comment:
I think this link should be `https://github.com/MediaBrowser/Emby/labels/up-for-grabs` - the `%20`s are turned into spaces, which means you don't get the right label.